src/main/java/io/seqera/tower/cli/responses/RunView.java

```diff
@@ -0,0 +1,128 @@
+package io.seqera.tower.cli.responses;
+
+import io.seqera.tower.cli.utils.TableList;
+import io.seqera.tower.model.Workflow;
+
+import java.io.PrintWriter;
+
+public class RunView extends Response {
```

Comment:
It will be interesting to add the web link to this run. You can use this:
http://github.com/seqeralabs/tower-cli/blob/f9ffc4d6d93e8d0948afffffcb1cb988b7f340ba/src/main/java/io/seqera/tower/cli/commands/LaunchCmd.java#L121-L128